string label
geometry_msgs/Pose2D[] SE2points
float64[] value
